Sources/App/configure.swift

```diff
@@ -11,15 +11,15 @@ enum ConfigurationError: Error {
 /// Called before your application initializes.
 public func configure(_ config: inout Config, _ env: inout Environment, _ services: inout Services) throws {
 
-    let logger = PrintLogger()
+    let logger = LogzIOLogger()
```

Comment:
Have we done this part?

> The log level should be also configurable through an environment variable to enable increasing/decreasing as needed without getting a new image. If not provided, INFO should be used as the default log level.

Reply:
Oh indeed missed that

Reply:
@dmcrodrigues Thinking about it, is this AC still relevant though, given that we get our logs via Logz.io anyway whose whole point is to provide nice ways for filtering things?

I mean if you want to filter logs, all you have to do is add a filter in the Kibana tab of Logz.io, no need to redeploy not even to change an env var in the manifest…

Reply:
I don't think there's a need to log all incoming requests, it's the only debug message that we have and why I think we should have info as the minimum log level.

Reply:
Ok. I'll address that tomorrow then 👍

Reply:
If it takes too much effort we can extract that part to a separate ticket, up to you.

Reply:
Implemented in 8c1280c805902432da9d36c20a7c398fee3daca8 + 9faf4f0afeabdf14abbadb469737e97196e33fa1